testng-core/src/main/java/org/testng/internal/TestNGClassFinder.java

```diff
@@ -5,10 +5,7 @@
 import java.lang.reflect.Constructor;
 import java.lang.reflect.Method;
 import java.lang.reflect.Modifier;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
I'm surprised that linter allows * import

Reply:
Not sure how that slipped through the linter. Fixed this by now adding explicitly all the imports.

Reply:
And even after adding that, the linter seems to be replacing it with the wild card imports. So I guess its the linter at work here :)